.pre-commit-config.yaml

```diff
@@ -7,3 +10,8 @@ repos:
     rev: 24.4.2
     hooks:
       - id: black
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: no-commit-to-branch
```

Comment:
Shall we specify the branch args?

Reply:
The default branch args is main and master